src/App.js

```diff
@@ -1,11 +1,28 @@
+import { Route, Routes } from "react-router-dom";
+import { AppLayout } from "./components/AppLayout";
+import { AuthScreen } from "./screens/AuthScreen";
+import { CartScreen } from "./screens/CartScreen";
+import { HomeScreen } from "./screens/HomeScreen";
+import { ProductsScreen } from "./screens/ProductsScreen";
+import { WishlistScreen } from "./screens/WishListScreen";
+import Mockman from "mockman-js";
```

Comment:
Within your screen directory, you can keep a index file, from where you can export all these screens, thus you would only have to do one import here, keeping your code even neat!

Reply:
Yes hamza. Thanks for pointing out. I'll refactor it in the next pr